Tests/NIOHTTP2Tests/ConfiguringPipelineAsyncMultiplexerTests.swift

```diff
@@ -61,7 +61,7 @@ final class ConfiguringPipelineAsyncMultiplexerTests: XCTestCase {
         typealias OutboundIn = HTTP2Frame.FramePayload
 
         func writeRequest(context: ChannelHandlerContext) {
-            context.channel.writeAndFlush(requestFramePayload, promise: nil)
+            context.writeAndFlush(NIOAny(requestFramePayload), promise: nil)
```

Comment:
Avoid using `NIOAny` within a handler, prefer using `self.wrapOutboundOut` when writing (and `self.wrapInboundOut` for `fireChannelRead`)